lib/public/utilities/ObservableData.js

```diff
@@ -111,6 +117,17 @@ class ObservableDataBuilder {
         return this;
     }
 
+    /**
+     * Any time the source data value change it will be reflected to the created observable data
```

Comment:
I think `reflected` is quite too ambiguous. 
Maybe:
Current value is updated with source's one when changed